composer.json

```diff
@@ -38,6 +38,9 @@
         "symfony/phpunit-bridge": "^4.3",
         "vimeo/psalm": "^3.4"
     },
+    "suggest": {
+        "monolog/monolog": "Allow sending log messages to sentry by using the included monolog handler."
```

Comment:
Can you please make the first letter of the `monolog` and `sentry` words uppercase?